### DISCUSSION TITLE
There is a way to hide the floating panel? (Not toggle it)

### DISCUSSION THREAD

**Discussion by danillos** [zellij-org/zellij#3202] [Q&A]
I would like to bind a key to always hide a floating panel, but I don't want to when I press it again, it shows the floating panel.

**Comment by danillos** [comment]
Another option, to solve my case, is a way to close the floating panel, but don't close a regular panel.

I would press a key, and it closes the floating panel, but if I press it again, I don't want it to close the regular panel.

**Comment by giyany** [comment]
Hi,

I'm not sure what your use case if. You can either toggle or close a floating pane: in the default, close it like any pane with Ctrl-p+x and toggle with Ctrl-p+f. Otherwise, there isn't a specific action to close only a floating pane (Zellij will close whatever pane is active).

**Reply by danillos** [reply]
Sorry, I forgot to say that I don't use the default Zellij workflow. I customized it for my personal productivity. So I don't have `Ctrl-p` to switch to panel mode.

Based on it, I guess it is a more personal issue than a general use case issue.

**Comment by imsnif** [comment]
You can do this with plugins (the interface is much more powerful).

**Reply by danillos** [reply]
Thank you for the insight. I'll check how plugins works.
